source/Calamari.Azure/Integration/Websites/Publishing/ServiceManagementPublishProfileProvider.cs

```diff
@@ -1,35 +1,50 @@
 ﻿using System;
 using System.Linq;
 using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using Calamari.Azure.Util;
 using Calamari.Commands.Support;
 using Microsoft.WindowsAzure;
 using Microsoft.WindowsAzure.Management.WebSites.Models;
+using Octopus.CoreUtilities.Extensions;
 
 namespace Calamari.Azure.Integration.Websites.Publishing
 {
     public class ServiceManagementPublishProfileProvider  
     {
         public static SitePublishProfile GetPublishProperties(string subscriptionId, byte[] certificateBytes, string siteName,string serviceManagementEndpoint)
         {
-            Log.Verbose($"servicemanagement endpoint is {serviceManagementEndpoint}");
-            using (var cloudClient = CloudContext.Clients.CreateWebSiteManagementClient(
-                new CertificateCloudCredentials(subscriptionId, new X509Certificate2(certificateBytes)),new Uri(serviceManagementEndpoint)))
+            try
```

Comment:
Ignore all the changes shown by the PR - the only actual change is wrapping it up int a try/catch block